File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-alert-triage-automation.ts
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
export const topic = "alert_triage";
export const fingerprint = "cce7a1ef9ec4b043";
export const alertLevel = (depth) => depth >= 24000 ? 'critical' : depth >= 12000 ? 'warning' : 'normal';

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-alert-triage-automation.py
"""Northstar Transit Cloud; Checkout SVC-17 incident context; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Checkout SVC-17"
INCIDENT = "INC-260713"
TOPIC = "alert_triage"
SPINE_IDS = ['org.ntc', 'svc.checkout.svc17', 'rel.inc260713.timeline', 'date.2026-07-20', 'term.no-kio-claim']
FINGERPRINT = "efa786b9c21f41a5"

def alert_level(queue_depth: int) -> str:
    return 'critical' if queue_depth >= 24000 else 'warning' if queue_depth >= 12000 else 'normal'
